modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -124,6 +128,7 @@
 
 			.e-n-tab-icon {
 				display: flex;
+			  	align-items: var( --n-tabs-icon-align-items );
```

Comment:
Possibly we can delete this and apply this as tab level.

Reply:
If I apply this to tab level the icon is not vertically centered.